### PR TITLE
Bundlesize2

### PR DESCRIPTION
**Pull request by robdodson**

### CODE REVIEW

**Comment by mihajlija** [comment]
@robdodson LGTM/PTAL

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by robdodson** [comment]
@mihajlija are you able to rebase against master? I want to make sure you get the final wording you want in the performance-budgets guide.

**Comment by mihajlija** [comment]
@robdodson I've rebased it, but now can't just push the branch because:

> error: failed to push some refs to 'https://github.com/GoogleChrome/web.dev.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.

What's the proper way to resolve this? Force push maybe? Or I can just bail on the rebase and resolve conflicts on merge with master?

**Comment by khempenius** [comment]
@robdodson As far as I'm aware there's no sort of caption styling/element for web.dev - could we add this? I think content/fast/incorporate-performance-budgets-into-your-build-tools/index.md is a good example of content that would benefit slightly from this.

**Comment by robdodson** [comment]
@mihajlija 	force pushing to the bundlesize2 branch will resolve it. The error shows up because when you rebase you change the history, so the remote branch complains. Force pushing to overwrite the remote branch is the correct way to resolve it.

@khempenius correct I don't think we currently have any styles for figcaptions. We could definitely add one. What do you think of something like this?

image

**Comment by mihajlija** [comment]
@robdodson @khempenius Re: images - I've been mostly using  tag instead of markdown/figure when I want to add the .screenshot class, but don't need a caption. Should I just use <figure> in that case in the future?

**Comment by robdodson** [comment]
ah right! if you need to add the `screenshot` class then you'll want to use `` as well.

But if it's just a regular uncaption image, and you don't need that class, then markdown is fine.

**Comment by mihajlija** [comment]
@robdodson @addyosmani LGTM/PTAL

**Comment by robdodson** [comment]
I'll take a look.

By the way @mihajlija, I was just going through and cleaning up the collaborators who have write access to this repo and I accidentally removed you. I added you back but I think it sent you another invite email that you'll need to click accept on.

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: content/fast/using-bundlesize-with-travis-ci/index.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ repository](https://help.github.com/articles/about-required-status-checks/) so
 that it won’t allow merging pull-requests unless the bundlesize tests have
 passed.
 
-<img src="bundlesize-check.jpg" class="screenshot" alt="Bundlesize check on GitHub">
+Bundlesize checks on GitHub include a size comparisson to the master branch and
```
Comment (robdodson) [inline content/fast/using-bundlesize-with-travis-ci/index.md]:
```suggestion
Bundlesize checks on GitHub include a size comparison to the master branch and
```

**Comment by robdodson** [comment]
@addyosmani 	@housseindjirdeh 	@khempenius 	are one of y'all free to give a look?

**Comment by khempenius** [comment]
LGTM. :shipit:

**Pull request merged**